lib/meta-box-partial-page.php

```diff
@@ -144,39 +147,70 @@ function gutenberg_meta_box_partial_page( $post_type, $meta_box_context ) {
  * @param WP_Locale $wp_locale      Locale object.
  */
 function gutenberg_meta_box_partial_page_admin_header( $hook_suffix, $current_screen, $wp_locale ) {
-	/* Scripts and styles that meta boxes can potentially be using */
-	wp_enqueue_style( 'common' );
-	wp_enqueue_style( 'buttons' );
-	wp_enqueue_style( 'colors' );
-	wp_enqueue_style( 'ie' );
-
+	/* Scripts that meta boxes can potentially be using */
 	wp_enqueue_script( 'utils' );
 	wp_enqueue_script( 'common' );
 	wp_enqueue_script( 'svg-painter' );
+	?>
+	<!-- Add an html class so that scroll bars can be removed in css and make it appear as though the iframe is one with Gutenberg. -->
+	<html class="gutenberg-meta-box-html">
+	<head>
+	<?php
+	// Grab the admin body class.
+	$admin_body_class = preg_replace( '/[^a-z0-9_-]+/i', '-', $hook_suffix );
 
-	// These assets are Gutenberg specific.
-	wp_enqueue_style(
-		'meta-box-gutenberg',
-		gutenberg_url( 'editor/build/meta-box-iframe.css' ),
-		array(),
-		filemtime( gutenberg_dir_path() . 'editor/build/meta-box-iframe.css' )
-	);
+	/**
+	 * The main way post.php sets body class.
+	 */
+	if ( get_user_setting( 'mfold' ) == 'f' ) {
+		$admin_body_class .= ' folded';
+	}
 
-	wp_enqueue_script(
-		'meta-box-resize',
-		gutenberg_url( 'assets/js/meta-box-resize.js' ),
-		array(),
-		filemtime( gutenberg_dir_path() . 'assets/js/meta-box-resize.js' ),
-		true
-	);
+	if ( ! get_user_setting( 'unfold' ) ) {
+		$admin_body_class .= ' auto-fold';
+	}
 
-	// Grab the admin body class.
-	$admin_body_class = preg_replace( '/[^a-z0-9_-]+/i', '-', $hook_suffix );
+	if ( is_admin_bar_showing() ) {
+		$admin_body_class .= ' admin-bar';
+	}
+
+	if ( is_rtl() ) {
+		$admin_body_class .= ' rtl';
+	}
+
+	if ( $current_screen->post_type ) {
+		$admin_body_class .= ' post-type-' . $current_screen->post_type;
+	}
+
+	if ( $current_screen->taxonomy ) {
+		$admin_body_class .= ' taxonomy-' . $current_screen->taxonomy;
+	}
+
+	$admin_body_class .= ' branch-' . str_replace( array( '.', ',' ), '-', floatval( get_bloginfo( 'version' ) ) );
+	$admin_body_class .= ' version-' . str_replace( '.', '-', preg_replace( '/^([.0-9]+).*/', '$1', get_bloginfo( 'version' ) ) );
+	$admin_body_class .= ' admin-color-' . sanitize_html_class( get_user_option( 'admin_color' ), 'fresh' );
+	$admin_body_class .= ' locale-' . sanitize_html_class( strtolower( str_replace( '_', '-', get_user_locale() ) ) );
+
+	if ( wp_is_mobile() ) {
+		$admin_body_class .= ' mobile';
+	}
+
+	if ( is_multisite() ) {
+		$admin_body_class .= ' multisite';
+	}
+
+	if ( is_network_admin() ) {
+		$admin_body_class .= ' network-admin';
+	}
+
+	$admin_body_class .= ' no-customize-support no-svg';
 
 	?>
-	<!-- Add an html class so that scroll bars can be removed in css and make it appear as though the iframe is one with Gutenberg. -->
-	<html class="gutenberg-meta-box-html">
-	<head>
+	</head>
+	<body class="wp-admin wp-core-ui no-js <?php echo $admin_body_classes . ' ' . $admin_body_class; ?>">
```

Comment:
Missing escaping

Reply:
This is the same code as the default admin_header. The idea is to have the same result. We may also remove it, I think it may be useless

Reply:
As a security measure, `esc_attr` should have no effect here if that's the case, unless an attack is in progress. Instead of trusting it's safe, lets eliminate all doubt and move forward with absolute rock solid certainty by escaping properly.